src/main/java/org/springframework/data/jpa/repository/support/SimpleJpaRepository.java

```diff
@@ -218,6 +218,16 @@ public void deleteAll(Iterable<? extends T> entities) {
 		}
 	}
 
+	@Override
+	public void deleteAllById(Iterable<? extends ID> ids) {
```

Comment:
Should we also add a `deleteAllByIdInBatch` variant?

Reply:
I added `deleteAllByIdInBatch`.

It does not work with EclipseLink since their handling of IN queries seems still to be broken.

Renamed `deleteInBatch` to `deleteAllInBatch` to match the naming conventions of `CrudRepository` and the new method. The old variant is deprecated and delegates to the new one.